libs/platform/src/lib/components/dynamic-page/dynamic-page-header/actions/global-actions/dynamic-page-global-actions.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, ChangeDetectionStrategy, ElementRef } from '@angular/core';
+
+@Component({
+    selector: 'fdp-dynamic-page-global-actions',
+    template: '<ng-content></ng-content>',
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
let's remove encapsulation, by adding ViewEncapsulation.None

Reply:
added `ViewEncapsulation.None` to this and other files